cellprofiler/preferences.py

```diff
@@ -1840,6 +1859,21 @@ def set_choose_image_set_frame_size(w, h):
     config_write(CHOOSE_IMAGE_SET_FRAME_SIZE, "%d,%d" % (w, h))
 
 
+__normalization_factor = "1.0"
+
+
+def get_normalization_factor():
```

Comment:
It might be a good idea to refactor the preferences here to be a class with `classmethod` properties (that seems like it should be possible at least). It also looks like there are a lot of other opportunities for refactoring in `preferences.py` :smile:

Reply:
So many opportunities! These can be tackled later.